src/screens/proposal.tsx

```diff
@@ -21,9 +43,88 @@ export const ProposalScreen = () => {
     app.removeListener('wallet-selected-proposal-deposit', onDepositSubmit);
   };
 
+  const onDeposit = () => {
+    showModal('wallets-bottom-sheet', {
+      wallets: visible,
+      closeDistance,
```

Comment:
why it there?

Reply:
Since the wallets modal window may not be large enough to close it with a swipe gesture, I decided to pass as a prop the value of the required swipe distance to hide

And ‘wallets.visible’ I think I need to request in modal